### PR TITLE
cvode: update to 6.0.0

### PR DESCRIPTION
**Pull request by neheb**
Build with ninja for speed.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: mingw-w64-cvode/PKGBUILD [hunk 1]
```diff
@@ -3,23 +3,17 @@
...
 pkgrel=1
 pkgdesc="ordinary differential equation system solver (mingw-w64)"
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url="https://computation.llnl.gov/projects/sundials"
 license=('BSD')
-makedepends=("${MINGW_PACKAGE_PREFIX}-cmake")
-source=("${_realname}-${pkgver}.tar.gz"::https://computation.llnl.gov/projects/sundials/download/${_realname}-${pkgver}.tar.gz
-        001-install-location.patch)
```
Comment (lazka) [inline mingw-w64-cvode/PKGBUILD:15]: I think this patch is still needed. All the DLLs are in the wrong place with this PR (see the CI logs)

**Comment by Biswa96** [comment]
The project url now redirects to a new one.

File: mingw-w64-cvode/PKGBUILD [hunk 2]
```diff
@@ -3,23 +3,17 @@
...
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=5.1.0
+pkgver=6.0.0
 pkgrel=1
 pkgdesc="ordinary differential equation system solver (mingw-w64)"
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url="https://computation.llnl.gov/projects/sundials"
```
Comment (Biswa96) [inline mingw-w64-cvode/PKGBUILD:11]:
```suggestion
url="https://computing.llnl.gov/projects/sundials/cvode"
```

File: mingw-w64-cvode/PKGBUILD (hunk 1 above)
Comment (neheb) [reply mingw-w64-cvode/PKGBUILD:15]: That patch didn't apply cleanly. Looking at upstream the cmake files have changed quite a bit. I'll look into it.

Comment (neheb) [reply mingw-w64-cvode/PKGBUILD:15]: Maybe now.

**Pull request merged**